### PR TITLE
UI Common Build Fixes

### PR DESCRIPTION
**Pull request by agiron123**
Changes to fix build issues with aqueducthq/common. Also updates version number to 0.0.3.

Related to:
https://linear.app/aqueducthq/issue/ENG-1109/refactor-ui-enterprise-to-import-from-ui-components-package

### CODE REVIEW

File: src/ui/common/src/components/workflows/edges/AqueductQuadratic.tsx [hunk 1]
```diff
@@ -1,23 +1,8 @@
 import React from 'react';
+import { EdgeProps } from 'react-flow-renderer';
 import {theme} from "../../../styles/theme/theme";
 
-type AqueductQuadraticProps = {
```
Comment (agiron123) [inline src/ui/common/src/components/workflows/edges/AqueductQuadratic.tsx:4]: These are no longer needed and we can just use the built in EdgeProps type that comes with react-flow.

File: src/ui/common/src/components/workflows/edges/AqueductBezier.tsx [hunk 2]
```diff
@@ -1,24 +1,8 @@
 import React from 'react';
-import { getBezierPath, Position } from 'react-flow-renderer';
-import {theme} from "../../../styles/theme/theme";
+import { EdgeProps, getBezierPath } from 'react-flow-renderer';
+import { theme } from "../../../styles/theme/theme";
 
-type AqueductBezierProps = {
```
Comment (agiron123) [inline src/ui/common/src/components/workflows/edges/AqueductBezier.tsx:5]: These are no longer needed and we can just use the built in EdgeProps type that comes with react-flow.

File: src/ui/common/src/components/workflows/edges/AqueductStraight.tsx [hunk 3]
```diff
@@ -1,23 +1,8 @@
 import React from 'react';
+import { EdgeProps } from 'react-flow-renderer';
 import {theme} from "../../../styles/theme/theme";
 
-type AqueductStraightProps = {
```
Comment (agiron123) [inline src/ui/common/src/components/workflows/edges/AqueductStraight.tsx:4]: These are no longer needed and we can just use the built in EdgeProps type that comes with react-flow.

File: src/ui/common/src/components/workflows/SideSheets/OperatorResultsSideSheet.tsx [hunk 4]
```diff
@@ -25,7 +25,6 @@ import {
     RelationalDBExtractParams,
     RelationalDBLoadParams
 } from "../../../utils/operators";
-import { CopyBlock } from "react-code-blocks";
```
Comment (agiron123) [inline src/ui/common/src/components/workflows/SideSheets/OperatorResultsSideSheet.tsx:28]: CopyBlock is giving us some trouble with building, so removing it for now.

We should revisit and find another component to use for the code blocks/copying in another task.

@vsreekanti  I noticed that there is a related PR here:
https://github.com/aqueducthq/enterprise/pull/521

Perhaps @Boyuan-Deng can take this on as some future work?

File: src/ui/app/package-lock.json [hunk 5]
```diff
@@ -1,9411 +0,0 @@
-{
```
Comment (vsreekanti) [inline src/ui/app/package-lock.json:1]: Was this intentionally removed @agiron123?

**Review by vsreekanti — APPROVED** [review]

File: src/ui/app/package-lock.json (hunk 5 above)
Comment (agiron123) [reply src/ui/app/package-lock.json:1]: I took this out to get fresh packages, but I expected it to be re-generated again for me. When installing via npm link apparently this does not get generated.

I will go ahead and publish the common package and re-generate this file before merging.

Comment (agiron123) [reply src/ui/app/package-lock.json]: So there was some weirdness with the package-lock being regenerated properly and what not. 

Instead there is another package to use, according to this github thread:
https://github.com/rogeriochaves/npm-force-resolutions/issues/36

Things are now working properly and this package is now used in both common and app.

**Pull request merged**